ibis/expr/types/geospatial.py

```diff
@@ -467,6 +989,10 @@ def end_point(self) -> PointValue:
         -------
         PointValue
             End point
+
+        Example
+        -------
+        ?? this only applies to lines and no lines in t.geom
```

Comment:
This method is specific too `LineStrings`, and we don't have a way to create one from the table to get an example

Reply:
Feel free to use as literal for anything that's specific to a certain type of geometry.